src/rpcblockchain.cpp

```diff
@@ -395,12 +395,15 @@ UniValue getblock(const UniValue& params, bool fHelp)
     if (fHelp || params.size() < 1 || params.size() > 2)
         throw runtime_error(
             "getblock \"hash|height\" ( verbose )\n"
```

Comment:
`s/verbose/verbosity/` (lost during the backport). Please rebase this change into the first commit, or into an intermediate commit (so the two together correspond to upstream's commit, and the final commit is our further changes).

Reply:
Done